packages/react-aria-components/stories/DatePicker.stories.tsx

```diff
@@ -52,6 +52,40 @@ export const DatePickerExample = () => (
   </DatePicker>
 );
 
+export const DatePickerTriggerWidthExample = () => (
+  <DatePicker data-testid="date-picker-example">
+    <Label style={{display: 'block'}}>Date</Label>
+    <Group style={{display: 'inline-flex', width: 300}}>
+      <DateInput className={styles.field} style={{flex: 1}}>
+        {segment => <DateSegment segment={segment} className={clsx(styles.segment, {[styles.placeholder]: segment.isPlaceholder})} />}
+      </DateInput>
+      <Button>🗓</Button>
+    </Group>
+    <Popover
+      placement="bottom start"
+      style={{
+        background: 'Canvas',
+        color: 'CanvasText',
+        border: '1px solid gray',
+        padding: 20,
+        width: 'calc(var(--trigger-width) - 20px * 2 - 1px * 2)'
```

Comment:
A bit unfortunate that the other popover styles force a calc to be a needed haha